data/db/misc.xml

```diff
@@ -483,4 +483,21 @@
         </calibration>
     </lens>
 
+    <lens>
+        <maker>Miranda</maker>
```

Comment:
I wasn't totally sure where to add this. The internet suggests that this might be the same as the Cosina 28mm f/2.8, but I'm not sure.

Reply:
Let's take what it says on the lens housing.  Apparently Miranda, accorgings to the uploader …